action/index.js

```diff
@@ -0,0 +1,50 @@
+const core = require('@actions/core');
+const github = require('@actions/github');
+const child_process = require('child_process');
+const fs = require('fs');
+
+/**
+ * Dump contents of file to console.
+ * @param {string} f - file to dump
+ */
+function dump(f) {
```

Comment:
- `dump` seems more like `print` where we are printing a file. I don't think `dump` is the correct technical term here.
- It's not common to use one letter variable names like Go. I'd assume something like `path`. I think we're accepting a path to a file, not the file itself.